src/app/redux/TransactionReducer.js

```diff
@@ -56,11 +56,27 @@ export default function reducer(state = defaultState, action) {
 
             let errorStr = error.toString();
             let errorKey = 'Transaction broadcast error.';
-            for (const [type /*, operation*/] of operations) {
+            for (const [type, operation] of operations) {
                 switch (type) {
                     case 'transfer':
                         if (/get_balance/.test(errorStr)) {
                             errorKey = 'Insufficient balance.';
+                        } else {
+                            for (
+                                let ei = 0;
+                                ei < error.data.stack.length;
+                                ei += 1
+                            ) {
+                                const errorStackItem = error.data.stack[ei];
+
+                                if (
+                                    errorStackItem.data.name === operation.to &&
+                                    errorStackItem.data.what === 'unknown key'
+                                ) {
+                                    errorKey = 'Unknown recipient';
```

Comment:
Ideally this would be an internationalized string.

Reply:
@gl2748 yea, it took me a while to compile the wallet from the train cuz the internet connection was crap or else I would have had more time to translate all the static texts from this file.